x/paloma/ante.go

```diff
@@ -78,7 +78,7 @@ func NewVerifyAuthorisedSignatureDecorator(fk types.FeegrantKeeper) VerifyAuthor
 // AnteHandle verifies that the message is signed by at least one signature that has
 // active fee grant from the creator address, IF the message contains metadata.
 func (d VerifyAuthorisedSignatureDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool, next sdk.AnteHandler) (sdk.Context, error) {
-	if simulate || ctx.IsCheckTx() {
```

Comment:
Why this change?

Reply:
@aleem1314 https://github.com/palomachain/paloma/pull/1040

Reply:
@aleem1314 sorry as i made a mistake by raising a pr to master branch and then changing it to v.50 upgrade branch this commit got pulled into this pr.